### DISCUSSION TITLE
Define or parse total of invoice and make a custom list

### DISCUSSION THREAD

**Discussion by jstorgaard** [jonaswinkler/paperless-ng#998] [Feature Requests]
I'm in a home renovation project and it would be nice to be able to generate lists to get a quick overview. 
I mark my documents as type (invoice) and correspondent and create a tag for room name and floor. That way I can filter out to get invoices from a specific correspondent or with items from a specific room.

My request is that I am able to generate a sort of list or summary of these invoices. Say total amount payed to a specific correspondent or total amount payed for a room or floor.
This list could include correspondent, filename, tag, total for each item.

**Comment by CrystalMethod** [comment]
relates to #1009?
